plugins/com.google.cloud.tools.eclipse.appengine.deploy.ui/src/com/google/cloud/tools/eclipse/appengine/deploy/ui/StandardDeployPreferencesPanel.java

```diff
@@ -187,7 +188,7 @@ private void setupProjectIdDataBinding(DataBindingContext context) {
         ViewerProperties.singleSelection().observe(projectSelector.getViewer());
     IObservableValue projectIdModel = PojoProperties.value("projectId").observe(model);
 
-    IValidator validator = new ProjectSelectorValidator(requireValues);
+    IValidator validator = requireValues ? new ProjectSelectorValidator() : null;
     context.bindValue(projectList, projectIdModel,
                       new UpdateValueStrategy().setConverter(new GcpProjectToProjectIdConverter())
                           .setAfterConvertValidator(validator),
```

Comment:
It's OK if this is null? It would seem to make more sense to just mot set this in case it's null.